### ISSUE TITLE
`'user_error!': Unknown method 'info'` when List package or install unity

### ISSUE THREAD

**Issue by sunmou99** [DragonBox/u3d#432]
### Issue Checklist

- [x] I'm using the latest u3d version
- [x] I ran `u3d --help`
- [x] I read the [README](https://github.com/DragonBox/u3d/blob/master/README.md)
- [x] I made sure that a similar [issue](https://github.com/DragonBox/u3d/issues) doesn't exit

### Issue Description

List package cmd has error:
u3d 1.3.2
ruby 2.6 ~ 3.1
os: macos, windows
```
u3d available -p -t
Ignoring digest-crc-0.6.3 because its extensions are not built. Try: gem pristine digest-crc --version 0.6.3
Ignoring ffi-1.12.2 because its extensions are not built. Try: gem pristine ffi --version 1.12.2
Ignoring unf_ext-0.0.7.7 because its extensions are not built. Try: gem pristine unf_ext --version 0.0.7.7
/Library/Ruby/Gems/2.6.0/gems/u3d-1.3.2/lib/u3d_core/ui/interface.rb:171:in `user_error!': Unknown method 'info', supported [:password, :not_implemented, :header, :message, :success, :command, :input, :command_output, :interactive?, :crash!, :select, :important, :confirm, :deprecated, :error, :user_error!, :verbose] (U3dCore::Interface::UIError)
	from /Library/Ruby/Gems/2.6.0/gems/u3d-1.3.2/lib/u3d_core/ui/ui.rb:42:in `method_missing'
	from /Library/Ruby/Gems/2.6.0/gems/u3d-1.3.2/lib/u3d_core/ui/ui.rb:40:in `method_missing'
	from /Library/Ruby/Gems/2.6.0/gems/u3d-1.3.2/lib/u3d/hub_modules_parser.rb:57:in `load_modules'
	from /Library/Ruby/Gems/2.6.0/gems/u3d-1.3.2/lib/u3d/unity_module.rb:88:in `block in load_versions_modules'
	from /Library/Ruby/Gems/2.6.0/gems/u3d-1.3.2/lib/u3d/unity_module.rb:88:in `map'
	from /Library/Ruby/Gems/2.6.0/gems/u3d-1.3.2/lib/u3d/unity_module.rb:88:in `load_versions_modules'
	from /Library/Ruby/Gems/2.6.0/gems/u3d-1.3.2/lib/u3d/unity_module.rb:67:in `load_modules'
	from /Library/Ruby/Gems/2.6.0/gems/u3d-1.3.2/lib/u3d/commands.rb:129:in `list_available'
	from /Library/Ruby/Gems/2.6.0/gems/u3d-1.3.2/lib/u3d/commands_generator.rb:142:in `block (2 levels) in run'
	from /Library/Ruby/Gems/2.6.0/gems/commander-4.6.0/lib/commander/command.rb:187:in `call'
	from /Library/Ruby/Gems/2.6.0/gems/commander-4.6.0/lib/commander/command.rb:157:in `run'
	from /Library/Ruby/Gems/2.6.0/gems/commander-4.6.0/lib/commander/runner.rb:444:in `run_active_command'
	from /Library/Ruby/Gems/2.6.0/gems/commander-4.6.0/lib/commander/runner.rb:58:in `run!'
	from /Library/Ruby/Gems/2.6.0/gems/commander-4.6.0/lib/commander/delegates.rb:18:in `run!'
	from /Library/Ruby/Gems/2.6.0/gems/u3d-1.3.2/lib/u3d/commands_generator.rb:265:in `run'
	from /Library/Ruby/Gems/2.6.0/gems/u3d-1.3.2/lib/u3d/commands_generator.rb:43:in `start'
	from /Library/Ruby/Gems/2.6.0/gems/u3d-1.3.2/exe/u3d:9:in `<top (required)>'
	from /usr/local/bin/u3d:23:in `load'
	from /usr/local/bin/u3d:23:in `<main>'
```
```
u3d install 2020.3.34f1 --trace
C:/hostedtoolcache/windows/Ruby/3.1.2/x64/lib/ruby/gems/3.1.0/gems/u3d-1.3.2/lib/u3d_core/ui/interface.rb:171:in `user_error!': Unknown method 'info', supported [:important, :message, :confirm, :verbose, :select, :user_error!, :error, :command_output, :not_implemented, :success, :deprecated, :command, :password, :crash!, :interactive?, :header, :input] (U3dCore::Interface::UIError)
	from C:/hostedtoolcache/windows/Ruby/3.1.2/x64/lib/ruby/gems/3.1.0/gems/u3d-1.3.2/lib/u3d_core/ui/ui.rb:42:in `method_missing'
	from C:/hostedtoolcache/windows/Ruby/3.1.2/x64/lib/ruby/gems/3.1.0/gems/u3d-1.3.2/lib/u3d_core/ui/ui.rb:40:in `method_missing'
	from C:/hostedtoolcache/windows/Ruby/3.1.2/x64/lib/ruby/gems/3.1.0/gems/u3d-1.3.2/lib/u3d/hub_modules_parser.rb:57:in `load_modules'
	from C:/hostedtoolcache/windows/Ruby/3.1.2/x64/lib/ruby/gems/3.1.0/gems/u3d-1.3.2/lib/u3d/unity_module.rb:104:in `load_version_modules'
	from C:/hostedtoolcache/windows/Ruby/3.1.2/x64/lib/ruby/gems/3.1.0/gems/u3d-1.3.2/lib/u3d/unity_module.rb:70:in `load_modules'
	from C:/hostedtoolcache/windows/Ruby/3.1.2/x64/lib/ruby/gems/3.1.0/gems/u3d-1.3.2/lib/u3d/unity_version_definition.rb:43:in `initialize'
	from C:/hostedtoolcache/windows/Ruby/3.1.2/x64/lib/ruby/gems/3.1.0/gems/u3d-1.3.2/lib/u3d/commands.rb:155:in `new'
	from C:/hostedtoolcache/windows/Ruby/3.1.2/x64/lib/ruby/gems/3.1.0/gems/u3d-1.3.2/lib/u3d/commands.rb:155:in `install'
	from C:/hostedtoolcache/windows/Ruby/3.1.2/x64/lib/ruby/gems/3.1.0/gems/u3d-1.3.2/lib/u3d/commands_generator.rb:178:in `block (2 levels) in run'
	from C:/hostedtoolcache/windows/Ruby/3.1.2/x64/lib/ruby/gems/3.1.0/gems/commander-4.6.0/lib/commander/command.rb:187:in `call'
	from C:/hostedtoolcache/windows/Ruby/3.1.2/x64/lib/ruby/gems/3.1.0/gems/commander-4.6.0/lib/commander/command.rb:157:in `run'
	from C:/hostedtoolcache/windows/Ruby/3.1.2/x64/lib/ruby/gems/3.1.0/gems/commander-4.6.0/lib/commander/runner.rb:444:in `run_active_command'
	from C:/hostedtoolcache/windows/Ruby/3.1.2/x64/lib/ruby/gems/3.1.0/gems/commander-4.6.0/lib/commander/runner.rb:58:in `run!'
	from C:/hostedtoolcache/windows/Ruby/3.1.2/x64/lib/ruby/gems/3.1.0/gems/commander-4.6.0/lib/commander/delegates.rb:18:in `run!'
	from C:/hostedtoolcache/windows/Ruby/3.1.2/x64/lib/ruby/gems/3.1.0/gems/u3d-1.3.2/lib/u3d/commands_generator.rb:265:in `run'
	from C:/hostedtoolcache/windows/Ruby/3.1.2/x64/lib/ruby/gems/3.1.0/gems/u3d-1.3.2/lib/u3d/commands_generator.rb:43:in `start'
	from C:/hostedtoolcache/windows/Ruby/3.1.2/x64/lib/ruby/gems/3.1.0/gems/u3d-1.3.2/exe/u3d:9:in `<top (required)>'
	from C:/hostedtoolcache/windows/Ruby/3.1.2/x64/bin/u3d:32:in `load'
	from C:/hostedtoolcache/windows/Ruby/3.1.2/x64/bin/u3d:32:in `<main>'
```

**Comment by niezbop** [comment]
Fixed by #431 thanks to @jlsalmon

**Comment by niezbop** [comment]
It'll be part of the next release
